### DISCUSSION TITLE
Effects of GPU number

### DISCUSSION THREAD

**Discussion by Holmes-GU** [STVIR/pysot#437] [General] [answered]
I only have two 1080ti GPUS and I use the following commands to train SiamRPN++:
CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.launch --nproc_per_node=2  --master_port=2333 ../../tools/train.py --cfg config.yaml.
Whether the number of GUPs will have  huge effects on the final results?
Thank you very much.

**Comment by ZhiyuanChen** [comment, marked as answer]
The batch size is highly related to the number of GPUs, which has significant influence on the final results. So, Yes.
